extensions/2.0/Vendor/EXT_property_animation/README.md

```diff
@@ -0,0 +1,111 @@
+# EXT\_property\_animation
+
+## Contributors
+
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Lewis Weaver, Microsoft
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds a specification for animation data targeting arbitrary
+properties, such as material colors, texture transform matrices and
+extension properties.
+
+## Extending Animations
+
+Property animations can be added to an animation by adding the
+`EXT_property_animation` extension to any glTF animation. For example, the
+following defines an animation with two channels that modify a material's
+baseColorFactor and roughnessFactor.
+```json
+"animations" : [
+    {
+        "channels" : [], 
+        "extensions" : {
+            "EXT_property_animation" : {
+                "channels" : [
+                    {
+                        "sampler" : 0,
+                        "target" : "/materials/1/pbrMetallicRoughness/roughnessFactor"
```

Comment:
Why target a material in the top-level list, rather than an instance of that material? For example:

```
"target": {
  "node": 25,
  "path": "/mesh/primitives/0/material/pbrMetallicRoughness/roughnessFactor"
}
```

Reply:
I had this initially but the engines I looked at need to collect and clone the materials as if the path implies the channel should only effect the single mesh primitive and not others that use the same material reference. Top-level approach leaves the whole material targeted everywhere its used; if unanimated material is desired those primitives would need a separate material that isn't targeted. Plus there is a little complexity describing a transformed JSON as you indicate here.

Reply:
Ok, I've come around on this. 👍